bugbug/test_scheduling.py

```diff
@@ -125,11 +126,11 @@ def set_touched_together(f1, f2):
     key = get_touched_together_key(f1, f2)
 
     if key not in touched_together:
-        touched_together[key] = (1).to_bytes(4, sys.byteorder)
+        touched_together[key] = struct.pack("i", 1)
```

Comment:
It's always positive, so you can use an unsigned int (`I`).

Reply:
In the `else` is also always positive?

Reply:
Yep, it's the same number, in the else it's just increased by 1 :)